src/vs/workbench/contrib/positronPlots/browser/positronPlotsView.tsx

```diff
@@ -206,8 +195,31 @@ export class PositronPlotsViewPane extends PositronViewPane implements IReactCom
 		this._positronPlotsContainer = DOM.$('.positron-plots-container');
 		container.appendChild(this._positronPlotsContainer);
 
+		// Observe the plots container for resizes and fire size/position changed events.
```

Comment:
This fixes the layout when expanding/collapsing the plots pane or neighboring panes.